src/plugins/console/public/application/containers/editor/monaco/utils/constants.ts

```diff
@@ -13,6 +13,15 @@ import { i18n } from '@kbn/i18n';
  */
 export const SELECTED_REQUESTS_CLASSNAME = 'console__monaco_editor__selectedRequests';
 
+/*
+ * CSS class names used for the styling of multiple-response status codes
+ */
+export const PRIMARY_STATUS_BADGE_CLASSNAME = 'monaco__status_badge__success';
+export const SUCCESS_STATUS_BADGE_CLASSNAME = 'monaco__status_badge__success';
+export const DEFAULT_STATUS_BADGE_CLASSNAME = 'monaco__status_badge__success';
+export const WARNING_STATUS_BADGE_CLASSNAME = 'monaco__status_badge__warning';
+export const DANGER_STATUS_BADGE_CLASSNAME = 'monaco__status_badge__danger';
```

Comment:
I think there is a typo in the primary and default class names

Reply:
Nice catch! Fixed in [0538534](https://github.com/elastic/kibana/pull/189210/commits/0538534133858e47d7ef32a7674cf42a4d648ec7).